apps/vr-tests-react-components/src/stories/InfoButton.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { InfoButton } from '@fluentui/react-infobutton';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('InfoButton', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <StoryWright
+      steps={new Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .hover('.info-button')
+        .snapshot('hover', { cropTo: '.testWrapper' })
+        // keys 'tab' is used instead of .focus, since .focus won't show the focus indicator.
+        .keys('.info-button', 'Tab')
+        .snapshot('focus', { cropTo: '.testWrapper' })
+        .click('.info-button')
+        .snapshot('active', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </StoryWright>
+  ))
+  .addStory(
+    'default',
+    () => (
+      <div style={{ padding: '10px' }}>
+        <InfoButton className="info-button" content="This is the content of an InfoButton." />
+      </div>
+    ),
+    {
+      includeHighContrast: true,
+      includeDarkMode: true,
+    },
+  );
+
+storiesOf('InfoButton', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <StoryWright steps={new Steps().snapshot('rest', { cropTo: '.testWrapper' }).end()}>{story()}</StoryWright>
+  ))
+  .addStory(
+    'size',
+    () => (
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          paddingTop: '60px',
+          paddingLeft: '10px',
+          paddingBottom: '10px',
+          gap: '80px',
+          alignItems: 'start',
+        }}
+      >
+        <InfoButton size="small" content="This is the content of an InfoButton." popover={{ open: true }} />
+        <InfoButton size="medium" content="This is the content of an InfoButton." popover={{ open: true }} />
+        <InfoButton size="large" content="This is the content of an InfoButton." popover={{ open: true }} />
+      </div>
+    ),
+    {
+      includeHighContrast: true,
+      includeDarkMode: true,
+    },
```

Comment:
High contrast and dark mode are already effectively tested in the other set of stories. It's not really necessary to include them here IMO.